x-pack/plugin/security/cli/src/test/java/org/elasticsearch/xpack/security/cli/EnrollNodeToClusterTests.java

```diff
@@ -248,6 +288,12 @@ public void testEnrollmentExitsOnExistingTlsConfiguration() throws Exception {
         assertAutoConfigurationFilesNotCreated();
     }
 
+    public void testEnrollmentOverwritesPreviousAutoconfigurationWhenForced() throws Exception {
+        final List<String> vanillaFileLines = Files.readAllLines(getResourcePath("elasticsearch.yml"), StandardCharsets.UTF_8);
```

Comment:
The two files in question don't have to be the "real" ES config right? Since they are copies they could quickly get out of date. It seems like this test is really (or should be ) a unit test of removePreviousAutoconfiguration, which could use synthetic+dynamic data to assert the behavior.

Reply:
ok, gotcha, I'll remove this entirely and move this to a unit test in the CLI class tests.